src/main/scala/viper/gobra/frontend/info/implementation/typing/StmtTyping.scala

```diff
@@ -50,16 +51,16 @@ trait StmtTyping extends BaseTyping { this: TypeInfoImpl =>
         })
 
     case n@PAssignment(rights, lefts) =>
-      rights.flatMap(isExpr(_).out) ++ lefts.flatMap(isExpr(_).out) ++
+      rights.flatMap(isExpr(_).out) ++ lefts.flatMap(isExpr(_).out) ++ lefts.flatMap(isNotExclusiveGlobal) ++
```

Comment:
this should be caught by assingable (i.e. `isNotExclusiveGlobal` should not be there).